### PR TITLE
eclass/gstreamer-meson.eclass: fix gst

### PR DESCRIPTION
**Pull request by rufq**
fix symlinking issue

### CODE REVIEW

**Comment by thesamesam** [comment]
If possible, could you explain the changes/why they're needed within each commit? (Also, fix the first commit message to match the second's format).

**Review by rufq — COMMENTED** [review]
The previous problem was that anyone who symlinked their tmp build directory caused the substitution not work due to `plugin` not matching a `BUILD_DIR` pattern. This should resolve the issue.

**Review by rufq — COMMENTED** [review]
Decline this commit because it didn't solve the symlink issue.

File: eclass/gstreamer-meson.eclass [hunk 1]
```diff
@@ -187,6 +187,7 @@ BDEPEND="
 	>=sys-apps/sed-4
 	virtual/pkgconfig
 	virtual/perl-JSON-PP
+	sys-apps/coreutils
```
[the remark this answers is not in the document]
Comment (rufq) [reply eclass/gstreamer-meson.eclass:190]: This actually isn't needed; it can be removed.

**Comment by scardracs** [comment]
Please change the name with eclass/gstreamer-meson.eclass: fix gst. This helps other on understand what you are actually doing faster

File: eclass/gstreamer-meson.eclass (hunk 1 above)
[the remark this answers is not in the document]
Comment (scardracs) [reply eclass/gstreamer-meson.eclass:190]: Remove it on your side and merge commits in order to maintain it as ordered as possible

**Comment by rufq** [comment]
its one commit now

**Comment by rufq** [comment]
removed the two older ones

**Comment by scardracs** [comment]
Before I forgot, please order deps alphabetically

**Comment by thesamesam** [comment]
> Before I forgot, please order deps alphabetically

I generally don’t mess with GNOME / Gst ones given they have their own rules.

**Comment by rufq** [comment]
I don't think there are any dependencies to be listed. It uses the same packages that are already present in the eclass: no additions.

**Pull request closed without merging**